doc/api/buffer.markdown

```diff
@@ -541,6 +541,42 @@ console.log(bufA.length);
 // 42
 ```
 
+### Class Method: Buffer.encode(str[, str_encoding][, target_encoding])
```

Comment:
Both are technically optional but Buffer.encode(str) would simply write from utf8 to utf8 which is silly :-)